environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentOpDescription.java

```diff
@@ -25,6 +25,10 @@
     public static final String GET_CRN_BY_NAME = "Get the crn of an environment by name.";
     public static final String GET_NAME_BY_CRN = "Get the name of an environment by crn.";
     public static final String UPDATE_CONFIG_BY_CRN = "Update the configuration for all stacks in the Environment by the Environment CRN";
+    public static final String UPDATE_LOAD_BALANCERS_BY_ENV_NAME = "Updates all cluster in an environment with load balancers, including adding the " +
+        "Endpoint Access Gateway if it's enabled. Environment is specified by name.";
```

Comment:
Reword a bit.
Something like: "Updates all clusters in an env with load balancers, including adding the Endpoint Access Gateway by the Environment name."

Reply:
Technically that's grammatically incorrect. The "by the Environment name" clause is in the wrong place because it looks like it applies to the Endpoint Gateway and not the environment. And I don't think we get enough out of shortening "environment" to make it worth it. I believe this is customer facing information. Unless you have a really strong opinion I'd prefer to leave it as-is.